drive_cli/utils.py

```diff
@@ -369,10 +369,23 @@ def upload_file(name, path, pid):
         'parents': [pid],
         'mimeType': file_mimeType
     }
-    media = MediaFileUpload(path, mimetype=file_mimeType)
-    new_file = service.files().create(body=file_metadata,
-                                      media_body=media,
-                                      fields='id').execute()
+    if os.stat(path).st_size <= 256 * 1024:
+        media = MediaFileUpload(path, mimetype=file_mimeType)
+        new_file = service.files().create(body=file_metadata,
+                                          media_body=media,
+                                          fields='id').execute()
+    else:
+        CHUNK_SIZE_MB = 1  # MB. You may want to increase the size to a higher speed if the network restrictions allow
+        media = MediaFileUpload(
+            path, mimetype=file_mimeType,
+            chunksize=1024 * 1024 * 1, resumable=True)
```

Comment:
I think the `chunksize` argument should be `1024 * 1024 * CHUNK_SIZE_MB`.

Reply:
Right.
But since there is no way to change the size definition, can we give it up?
Maybe we'll add this as an ENV variable?

```suggestion
            CHUNK_SIZE_MB = int(os.getenv("CHUNK_SIZE_MB") or 1
            chunksize=(1024 * 1024 * CHUNK_SIZE_MB) , resumable=True)
```